demo/EXPLOR3R/auto-drive.py

```diff
@@ -64,9 +66,9 @@ def backup():
     Back away from an obstacle.
     """
 
-    # Sound backup alarm.
+    # Sound backup alarm. 
     Sound.tone([(1000, 500, 500)] * 3)
-
+    
```

Comment:
You're right, it has absolutely no effect on the code's functionality. I ask for those changes mainly because they make the diff easier to read and keep the file uniform. For example, if I find a new bug next week and want to use git's "blame" feature to figure out when this section of code was last modified, I would find that _you_ were the last one to change the code, even though all you did was add some whitespace. Keeping those extraneous changes out of our commit history makes things like that significantly easier.